source/ui/BioXAS/BioXASControlEditor.cpp

```diff
@@ -262,6 +291,32 @@ void BioXASControlEditor::updateUnits()
 		BioXASValueEditor::setUnits(control_->units());
 }
 
+void BioXASControlEditor::updateProgressValueMinimum()
+{
+	if (control_ && useControlMovingAsProgress_ && control_->moveInProgress())
+		BioXASValueEditor::setProgressValueMinimum(control_->value());
+}
+
+void BioXASControlEditor::updateProgressValueMaximum()
+{
+	if (control_ && useControlMovingAsProgress_ && control_->moveInProgress())
+		BioXASValueEditor::setProgressValueMaximum(control_->setpoint());
+}
+
+void BioXASControlEditor::updateProgressValue()
+{
+	if (control_ && useControlMovingAsProgress_ && control_->moveInProgress())
+		BioXASValueEditor::setProgressValue(control_->value());
```

Comment:
Why do you need to specifically scope the BioXASValueEditor?  I didn't notice any inheritance or conflicts in your changes.